PRIVATE_SYNTAX_FAQ.md

```diff
@@ -80,6 +102,22 @@ This would work, but would be an [ASI hazard](https://github.com/tc39/proposal-p
 
 More generally, the committee has mostly come down on the side of `this.#` as more clearly implying field access. 
 
+## Why aren't declarations `private x`?
+
+This sort of declaration is what other languages use (notably Java), and implies that access would be done with `this.x`. Assuming that isn't the case (see above), in JavaScript this would silently create or access a public field, rather than throwing an error. This is a major potential source of bugs or invisibly making public fields which were intended to be private.
+
+It also allows a symmetry between declaration and access, just as there is for public fields:
```

Comment:
I think t would help this item if we had an example of accessing a field on another object (not `this`)